### PR TITLE
l10n: Remove a text string from translation

### PR DESCRIPTION
**Pull request by Valdnet**
Removal of a text string from a translation in Transifex.

Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

**Review by rakekniven — APPROVED** [review]

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Review by rakekniven — COMMENTED** [review]
I will stop reviewing such things without being a coder ;-)

File: apps/files_sharing/templates/public.php [hunk 1]
```diff
@@ -75,7 +75,7 @@
 						<?php if (isset($_['mimetype']) && strpos($_['mimetype'], 'image') === 0) { ?>
 							<div class="directDownload">
 								<div>
-									<?php p($l->t('%s', [$_['filename']]))?> (<?php p($_['fileSize']) ?>)
+									<?php p('%s', [$_['filename']])?> (<?php p($_['fileSize']) ?>)
```
[the remark this answers is not in the document]
Comment (Valdnet) [reply apps/files_sharing/templates/public.php]: Hmmm… Was this code badly written from scratch? I am asking you to know for the future 😉.

Code corrected 😁.

**Comment by Valdnet** [comment]
> I will stop reviewing such things without being a coder ;-)

@rakekniven I'm not a programmer either. I did some coding there, but we can always rely on developers like nickvergessen.

**Comment by rakekniven** [comment]
> > I will stop reviewing such things without being a coder ;-)
> 
> @rakekniven I'm not a programmer either. I did some coding there, but we can always rely on developers like nickvergessen.

Yes, that is fine for me.

In my case I approved your change as a reviewer and that was wrong.

File: apps/files_sharing/templates/public.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (Shar1988) [reply apps/files_sharing/templates/public.php]: https://github.com/nextcloud/server.git

File: apps/files_sharing/templates/public.php [hunk 2]
```diff
@@ -75,7 +75,7 @@
 						<?php if (isset($_['mimetype']) && strpos($_['mimetype'], 'image') === 0) { ?>
 							<div class="directDownload">
 								<div>
```
Comment (Shar1988) [inline apps/files_sharing/templates/public.php:77]: _`

File: apps/files_sharing/templates/public.php [hunk 3]
```diff
@@ -87,7 +87,7 @@
 				<?php if ($_['previewURL'] === $_['downloadURL'] && !$_['hideDownload']): ?>
```
Comment (Shar1988) [inline apps/files_sharing/templates/public.php:87]: ?

File: apps/files_sharing/templates/public.php (hunk 2 above)
Comment (Shar1988) [inline apps/files_sharing/templates/public.php:77]: div

Comment (Shar1988) [inline apps/files_sharing/templates/public.php:77]: facebook.com

File: apps/files_sharing/templates/public.php [hunk 4]
```diff
@@ -75,7 +75,7 @@
 						<?php if (isset($_['mimetype']) && strpos($_['mimetype'], 'image') === 0) { ?>
 							<div class="directDownload">
```
Comment (Shar1988) [inline apps/files_sharing/templates/public.php:76]: google.com

File: apps/files_sharing/templates/public.php [hunk 5]
```diff
@@ -87,7 +87,7 @@
 				<?php if ($_['previewURL'] === $_['downloadURL'] && !$_['hideDownload']): ?>
 					<div class="directDownload">
 						<div>
-							<?php p($l->t('%s', [$_['filename']]))?> (<?php p($_['fileSize']) ?>)
+							<?php p($_['filename'])?> (<?php p($_['fileSize']) ?>)
```
Comment (Shar1988) [inline apps/files_sharing/templates/public.php:90]: google.com

File: apps/files_sharing/templates/public.php [hunk 6]
```diff
@@ -75,7 +75,7 @@
 						<?php if (isset($_['mimetype']) && strpos($_['mimetype'], 'image') === 0) { ?>
```
Comment (Shar1988) [inline apps/files_sharing/templates/public.php:75]: if

**Review by Shar1988 — APPROVED** [review]

**Review by Shar1988 — APPROVED** [review]

**Review by Shar1988 — APPROVED** [review]

**Review by Shar1988 — APPROVED** [review]

**Review by szaimen — APPROVED** [review]
LGTM

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**